go/caddy/pfpki/cloud/intune.go

```diff
@@ -161,10 +164,31 @@ func (cl *Intune) NewCloud(ctx context.Context, name string) error {
 	client := &http.Client{Transport: tr}
 	cl.Client = client
 
-	req, err := http.NewRequest("GET", graphRequest, nil)
+	var aadGraphFailed bool
```

Comment:
it should be msGraphFailed instead since the first request is for msGraphResourceUrl